deploy/helm/values.schema.json

```
@@ -250,6 +250,13 @@
                 "type": "string"
             }
         },
+        "storageClassName": {
+            "description": "Kubernetes storage classes. More info: https://kubernetes.io/docs/concepts/storage/storage-classes",
+            "type": "object",
+            "additionalProperties": {
+               "type": "string"
+            }
+        },
```

We add `k8s` prefix to definitions, that are defined by Kubernetes. Also, the definition itself seems wrong. The `storageClassName` is an optional string, not an objet.
```suggestion
        "k8sStorageClassName": {
            "description": "The name of the StorageClass required by the claim. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class",
            "type": [
                "string",
                "null"
            ]
        },
```